kloppy/infra/serializers/event/wyscout/deserializer_v2.py

```diff
@@ -477,6 +510,24 @@ def deserialize(self, inputs: WyscoutInputs) -> EventDataset:
                         **generic_event_args,
                     )
 
+                # If also an interception event, add this before other event (It is a tag on multiple wyscout_events)
```

Comment:
Wyscout does not have a separate event type for interceptions. Instead, interceptions are recorded as a pass, duel or touch event with an "interception" tag. Your current implementation works well for passes, but for consistency with other providers (and socceraction) I would convert these duels/touch events to an interception. Not to an interception + duel.